plugins/win-dshow/virtualcam-module/placeholder.cpp

```diff
@@ -82,19 +83,32 @@ static bool load_placeholder_internal()
 {
 	Status s;
 
-	wchar_t file[MAX_PATH];
-	if (!GetModuleFileNameW(dll_inst, file, MAX_PATH)) {
-		return false;
-	}
+	wchar_t file[MAX_PATH] = {'\0'};
+	PWSTR pszPath = NULL;
+
+	HRESULT hr = SHGetKnownFolderPath(FOLDERID_RoamingAppData,
+					  KF_FLAG_DEFAULT, NULL, &pszPath);
+	if (hr == S_OK) {
```

Comment:
This if condition will always be true on any Windows system where AppData\Roaming exists.

Reply:
That's true, it will. That's what it's supposed to do.

Reply:
I think see you point now. My original code was clean and worked. I hacked up load_placeholder_internal as requested, and now it no longer does what was intended.

Reply:
The original code was the best solution. There are multiple checks that have to be performed on the png file in order to determine whether or not the custom png file can be used. 

My mistake was trying to change the code to get past the review.